paymentsheet/src/test/java/com/stripe/android/paymentsheet/EditPage.kt

```diff
@@ -28,15 +30,16 @@ internal class EditPage(
             .performClick()
     }
 
-    fun assertNotInDropdown(cardBrand: String) {
+    fun assertInDropdownButDisabled(cardBrand: String) {
         // Click on the dropdown menu to expand it
         composeTestRule.onNodeWithTag(DROPDOWN_MENU_CLICKABLE_TEST_TAG)
             .performClick()
 
-        // Attempt to find the node with the specified cardBrand
-        // and assert that it does not exist
+        // Attempt to find the node with the specified cardBrand,
+        // assert that it is present (displayed) and disabled
         composeTestRule.onNodeWithTag("${TEST_TAG_DROP_DOWN_CHOICE}_$cardBrand")
-            .assertDoesNotExist()
+            .assertIsDisplayed()
+            .assertIsNotEnabled()
```

Comment:
Do we need to update other selectors to ensure it's still enabled?

Reply:
Added a check that CB is still enabled in the drop down.